code/modules/surgery/organs/internal/cyberimp/augments_eyes.dm

```diff
@@ -14,6 +14,23 @@
 	slot = ORGAN_SLOT_HUD
 	var/HUD_type = 0
 	var/HUD_trait = null
+	actions_types = list(/datum/action/item_action/toggle_hud)
+	var/toggledOn = TRUE
```

Comment:
`snake_case`, also put `action_types` above all newly declared vars, also auto-doc this var

Reply:
I'm not sure what you mean by snake case, is it like_this? Also I'm not sure what autodocing the var is, sorry.